@xen-orchestra/backups/_runners/_writers/_MixinRemoteWriter.mjs

```diff
@@ -113,13 +113,13 @@ export const MixinRemoteWriter = (BaseClass = Object) =>
       )
     }
 
-    _isAlreadyTransferred(timestamp) {
+    async _isAlreadyTransferred(timestamp) {
       const vmUuid = this._vmUuid
       const adapter = this._adapter
       const backupDir = getVmBackupDir(vmUuid)
       try {
         const actualMetadata = JSON.parse(
-          adapter._handler.readFile(`${backupDir}/${formatFilenameDate(timestamp)}.json`)
+          await adapter._handler.readFile(`/${backupDir}/${formatFilenameDate(timestamp)}.json`)
```

Comment:
Is the added leading slash in the readFile path normal ?

Reply:
the leading slash shouldn't impact (since path are from the root of the remote) , but it's less confusing without